packages/components/src/__actions__/Button/v3/Button.module.css

```diff
@@ -206,6 +230,12 @@
     --button-bg-color: rgba(var(--color-white-rgb), 0.1);
     --button-border-color: transparent;
   }
+
+  &:disabled {
```

Comment:
Small quirk of how this component is composed since we're using RAC button as the underlying technology, I'd recommend styling this instead with isDisabled class instead of the pseudo selector. This guarentee that this style will be applied when buttons internal states to isDisabled :)